### PR TITLE
MCR-1627 Implemented RSS Feed Importer

### PR DESCRIPTION
**Pull request by fluetze**

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 87219acaaab41931fb0c8bd06c3fe0d950e78d69! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/1652-pull_request-271

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on f76942900eed801fe64342597a63340a047bef18! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/1654-pull_request-271

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 7bb74d5e111eda8430e85b3d8a9d9e30d17006cb! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/1665-pull_request-271

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 1885bbdacbbce1b1531d24c60308f9cc4945e222! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/1671-pull_request-271

File: mycore-mods/pom.xml [hunk 1]
```diff
@@ -1,90 +1,102 @@
...
-      <artifactId>xercesImpl</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.ibm.icu</groupId>
-      <artifactId>icu4j</artifactId>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
-</project>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sebhofmann) [inline mycore-mods/pom.xml:1]: The indent seems to wrong or maybe the line ending ?

File: pom.xml [hunk 2]
```diff
@@ -925,6 +926,11 @@
         <version>58.2</version>
       </dependency>
       <dependency>
+        <groupId>com.rometools</groupId>
+        <artifactId>rome</artifactId>
+        <version>1.5.1</version>
```
Comment (sebhofmann) [inline pom.xml]: rome version 1.5.1 is to old and uses JDOM(1).
(https://github.com/rometools/rome/blob/v1.5.1/pom.xml)
You should switch to at least 1.6.0 which is using jdom2

File: mycore-mods/src/main/java/org/mycore/mods/MCRMODSCommands.java [hunk 3]
```diff
@@ -1,113 +1,120 @@
...
-        }
-    }
-
-    private static void saveAsMyCoReObject(String projectID, Element modsRoot) throws MCRActiveLinkException, MCRPersistenceException, MCRAccessException {
-        MCRObject mcrObject = MCRMODSWrapper.wrapMODSDocument(modsRoot, projectID);
-        mcrObject.setId(MCRObjectID.getNextFreeId(mcrObject.getId().getBase()));
-        MCRMetadataManager.create(mcrObject);
-    }
-}
+/*
```
Comment (sebhofmann) [inline mycore-mods/src/main/java/org/mycore/mods/MCRMODSCommands.java:1]: same like in pom.xml

File: mycore-mods/src/main/java/org/mycore/mods/rss/MCRRSSFeedImporter.java [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+
+    private String field2queryID;
+
+    private String importURI;
+
+    private String xsl2BuildNotificationMail;
+
+    private static final String STATUS_FLAG = "imported";
+
+    private static final String PROPERTY_MAIL_ADDRESS = "MCR.Mail.Address";
```
Comment (sebhofmann) [inline mycore-mods/src/main/java/org/mycore/mods/rss/MCRRSSFeedImporter.java:112]: MCR.Mail.Recipients is better here, but not sure.

**Review by sebhofmann — CHANGES_REQUESTED** [review]

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 19364cc945bb2771cabdc722756b0f1499b60ae7! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/1673-pull_request-271

**Comment by fluetze** [comment]
Code is a mess, withdraw this pull request

**Pull request closed without merging**